### PR TITLE
Add --allow-all flag

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

**Comment by piscisaureus** [comment]
Cool no-nonsense name.... You're not wondering if people might get confused and expect it to actually sudo?

File: src/flags.rs [hunk 1]
```diff
@@ -26,6 +26,7 @@ pub struct DenoFlags {
   pub allow_net: bool,
   pub allow_env: bool,
   pub allow_run: bool,
+  pub sudo: bool,
```
Comment (kevinkassimo) [inline src/flags.rs]: I think we don't really need to add this to `DenoFlags`?...

**Comment by ry** [comment]
Hm - they might...
This looks odd:
`sudo deno script.ts --sudo`

**Comment by ry** [comment]
`--allow` ?

**Comment by kevinkassimo** [comment]
`-A` sounds fine as a short flag (I did propose something similar in the past).
For the long flag I personally prefer `--allow-all` to match with other perm flags (`--root` `--su` all have other meanings and might be confusing...)

**Comment by piscisaureus** [comment]
--unsafe ?
--godmode 
--boss 
--unrestricted 
--super
--aaa

**Comment by kevinkassimo** [comment]
`--doom` (yikes)

**Comment by ry** [comment]
I think `--allow-all` is most consistent and clear. But `--unsafe` seems cooler.

**Comment by kevinkassimo** [comment]
I feel we could reserve `--unsafe` or `--unsafe-*` for new flags if we later want to introduce other new flags wrt security model changes?

**Comment by shekohex** [comment]
--yolo 😂

File: src/flags.rs [hunk 2]
```diff
@@ -127,6 +133,7 @@ pub fn set_flags(
   opts.optflag("", "allow-net", "Allow network access.");
   opts.optflag("", "allow-env", "Allow environment access.");
   opts.optflag("", "allow-run", "Allow running subprocesses.");
+  opts.optflag("A", "allow-all", "Allow all permissions");
```
Comment (KluVerKamp) [inline src/flags.rs]: OCD alert, missing a dot at the end of the line!

**Pull request merged**